ci/serverless/pipeline/004_integration-filter.conf

```diff
@@ -11,7 +11,7 @@ input {
 filter {
     elastic_integration {
         hosts => "${ES_ENDPOINT}"
-        api_key => "${LS_ROLE_API_KEY_ENCODED}"
+        api_key => "${INTEGRATION_API_KEY_ENCODED}"
```

Comment:
Nothing to do with this PR, but it is confusing that we require a base encoded `api_key` for this filter and not for the elasticsearch plugins, and still have the same name in the config...